src/actions/team.js

```diff
@@ -49,16 +49,20 @@ export function teamPatched(json) {
 }
 
 export function updateTeam(payload) {
+  const typedPayload = Object.assign({}, payload);
```

Comment:
Where should this go?  We need to ensure the string coming back from the text input field is actually an integer, and also snake-case the name to match the model.  I figured we wanted our database column to be `vote_duration` and not `voteDuration`, so this discrepancy needs to be resolved somewhere, unless we always want to refer to `vote_duration` in our state, etc.

Reply:
The API should definitely receive the field using snake case. If you ask me, though, this should probably be taken care of earlier than in the `updateTeam` action. My rationale is that the backend also uses some of these promises to transmit information, so they should probably be normalized to snake case before any actions are called on the data.

I would modify the `handleSubmit` method of `TeamForm` instead.

Reply:
Sounds good to me - I'll move it there.